src/index.html

```diff
@@ -4,6 +4,8 @@
         <title>Strangeluv</title>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="/tracking.js"></script>
```

Comment:
Have these pop on and off with the component.  Refer to the versions in node_modules, use minified.